File: src/main/java/tfg/proyecto/TFG/dtos/DTOInvitado.java
package tfg.proyecto.TFG.dtos;

import java.math.BigDecimal;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Builder
@Data
@NoArgsConstructor
@AllArgsConstructor
public class DTOInvitado {
	private String nombre;
    private String apellidos;
    private String fotoURL;
    private String descripcion;
}

File: frontend/appMovilShowPass/app/src/main/java/com/example/appmovilshowpass/data/remote/dto/DTOInvitado.kt
package com.example.appmovilshowpass.data.remote.dto

data class DTOInvitado(
    val id: Long,
    val nombre: String? = null,
    val apellidos: String? = null,
    val fotoURL: String? = null,
    val descripcion: String? = null
)